core/src/main/java/com/klaytn/caver/account/WeightedMultiSigOptions.java

```diff
@@ -43,6 +43,47 @@ public WeightedMultiSigOptions(BigInteger threshold, List<BigInteger> weights) {
         this.weights = weights;
     }
 
+
+    /**
+     * Create a WeightedMultiSigOptions instance fill with default value.(threshold, weight has 1)
+     * @param publicKeyArr public key array
+     * @return WeightedMultiSigOptions
+     */
+    public static WeightedMultiSigOptions fillWeightedMultiSigOptionForMultiSig(String[] publicKeyArr) {
```

Comment:
I think function name is different with what this function does.
This function does not fill anything.

Reply:
How about fillDefaultOptionsValue() for this method name?

Reply:
I renamed method fillDefaultValuesForWeightedMultSig and fillDefaultValuesForWeightedForRoleBased

Reply:
I think if you want to name the function with `fillxxx`, you have to get some target as a parameter of this function and fill in the default options here.

However, this is a function that returns default WeightedMultiSigOptions depending on the type of AccountKey, so what if we name it getDefaultOptionsForWeightedMultiSig? 

If i misunderstand this function, then please let me know.

Reply:
Okay, I modified this function name.